docs/src/algorithms/discretization.md

```diff
@@ -5,3 +5,53 @@ discretize
 DiscretizationMethod
 FIST
 ```
+# FIST  
+Examples for the FIST algorithm. 
+
+### Usage  
+  
+```@example overview
+# Import required Libraries
```

Comment:
Please assume that readers of this documentation are familiar with Julia basics in general. Comments are only useful when they bring something new to the table. For example, an insight. You will become a better contributor over time if you follow these basic principles :)

Reply:
Thank you, I will change this comment and similar ones.